spring-boot-starter/spring-boot-starter-registry/src/test/java/io/servicecomb/springboot/starter/registry/TestRegistryIntializer.java

```diff
@@ -0,0 +1,40 @@
+
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicecomb.springboot.starter.registry;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import io.servicecomb.serviceregistry.RegistryUtils;
+
+/**
+ *
+ */
+public class TestRegistryIntializer {
+
+  @BeforeClass
+  public static void beforeTest() {
+    System.setProperty("cse.rest.address", "127.0.0.1:8081");
+  }
+
+  @Test
+  public void testInitRegistry1() {
```

Comment:
I think this test case if covered by TestCseAutoConfiguration

Reply:
fixed